index.bs

```diff
@@ -349,7 +344,7 @@ spec: CSP; urlPrefix: https://www.w3.org/TR/CSP3/#
 
       1. Otherwise, [=set/for each=] <a>permissions-source-expression</a> |item| in the <a>allowlist</a>:
 
-        1. If the result of running <a>Does url match expression in origin with redirect count?</a>
+        1. If the result of running [=Does url match expression in origin with redirect count?=]
            on <var>origin</var>, |item|, <var>origin</var>, and 0 is true then return true.
```

Comment:
While you're in here, can you reword this so that we properly cast the first *origin* argument as a URL? You can close out #519 too, if you do that.